src/pigui/PiGuiSandbox.cpp

```diff
@@ -55,14 +55,18 @@ static int CleanupWindowStack(SavedImguiStackInfo *stackInfo)
 
 static int CleanupStyleStack(SavedImguiStackInfo *stackInfo)
 {
+	// `ImVector::size()` returns an `int`, and `stackInfo->style*Stack`
+	// properties are `uint32_t`. Therefore, in order to correctly perform
+	// comparison without triggering UB, the unsigned type must be casted
+	// to an bigger signed type. In this case, `uint32_t` -> `int64_t`.
 	auto &colorStack = ImGui::GetCurrentContext()->ColorModifiers;
 	int numResetStyles = colorStack.size() - stackInfo->styleColorStack;
-	if (colorStack.size() > static_cast<int64_t>(stackInfo->styleColorStack))
+	if (colorStack.size() > (int64_t)stackInfo->styleColorStack)
```

Comment:
```suggestion
	if (colorStack.size() > int64_t(stackInfo->styleColorStack))
```
This is a "functional style cast" - wrapping the parentheses around the cast type is a C-style cast. It's a stylistic decision that's used in the codebase.